R/MsBackendDataFrame.R

```diff
@@ -267,6 +261,17 @@ setMethod("msLevel", "MsBackendDataFrame", function(object, ...) {
     .get_column(object@spectraData, "msLevel")
 })
 
+#' @rdname hidden_aliases
+setReplaceMethod("msLevel", "MsBackendDataFrame", function(object, value) {
```

Comment:
Just wondering here - do we really want a replacement method for the MS level? Didn't we decided against it when designed to API?

Reply:
I see the `Spectra` object more as a core infrastructure object, so from that perspective I would let users set and change all spectra variables. A use case for setting the MS level are for example data files (including spectra databases) in which the MS level information is not provided. For these it might be helpful to actually define the MS level. It's similar to `centroided`, basically, this information should be retrieved from the mzML files, but e.g. mzXML or CDF does not provide this information.